packages/core/src/cache/cache.ts

```diff
@@ -17,13 +20,19 @@ const MAX_CACHE_SIZE_1GB = 1073741824;
 class Cache implements ICache {
   private readonly _imageCache: Map<string, ICachedImage>; // volatile space
   private readonly _volumeCache: Map<string, ICachedVolume>; // non-volatile space
+  // Todo: contour for now, but will be used for surface, etc.
+  private readonly _geometryCache: Map<string, ICachedGeometry>;
```

Comment:
Can this geometry cache be used to store the mapping to/from acquisition plane ranges?